Packs/AdminByRequest/Integrations/AdminByRequestEventCollector/AdminByRequestEventCollector_description.md

```diff
@@ -0,0 +1,8 @@
+### How to generate API key: 
+In order to  use the API , you must first enable the API and set up an API key.
+- Logging into your portal account.
+- Navigating to Settings -> Data -> API. 
+- You can use the copy icon to the right of the api key to copy it to the clipboard.
```

Comment:
- You can use the copy icon to the right of the API key to copy it to the clipboard.